atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -37,41 +37,123 @@
 
 /**
  * Provides the methods for a transaction with the key-value store.
+ *
+ * In general: users may assume that if maps (including sorted maps) keyed on byte[] are returned to the user,
+ * they may be accessed, including via random access, via any byte array that is equivalent in terms of
+ * {@link java.util.Arrays#equals(byte[], byte[])}.
+ *
  * @see TransactionManager
  */
 public interface Transaction {
 
+    /**
+     * Returns a mapping of rows to {@link RowResult}s within {@code tableRef} for the specified {@code rows}, loading
+     * columns according to the provided {@link ColumnSelection}.
+     *
+     * The returned {@link SortedMap} is sorted on the byte order of row keys; the ordering of the input parameter
```

Comment:
Sometimes you say "sorted on the byte order" and sometimes you add "with ascending byte ordering".

Reply:
I'll standardise to one of them